tests/test_content.py

```diff
@@ -352,3 +352,31 @@ def __format__(self, format):
     new_result = '{:open-the-pod-bay-doors}'.format(HAL9000())
 
     assert new_result == "I'm afraid I can't do that."  # output
+
+
+def test_side_note():
+    """
+    # Side note about new format
+
+    It can be a litle confusing when you have many string operation
+    like padding, aligning, truncating, etc. in the same string.
+
+    So a little note : the standard formatting is like this
+        `{index_0:>operation_0}{index_1:>operation_1} ...`
+    but you can ommited the `index` if it is in ascending order
+    """
+
+    old_result = '{:<10}'.format('one')
```

Comment:
The point of "old_result" is to work with the modulo-style formatters. Please don't abuse it like this :)